src/openMVG/localization/VoctreeLocalizer.cpp

```diff
@@ -589,7 +588,11 @@ bool VoctreeLocalizer::localizeFirstBestResult(const features::SIFT_Regions &que
                                                        true /*b_refine_pose*/, 
                                                        param._refineIntrinsics /*b_refine_intrinsic*/);
     if(!refineStatus)
-      POPART_COUT("[poseEstimation]\tRefine pose could not improve the estimation of the camera pose.");
+    {
+      POPART_COUT("[poseEstimation]\tRefine pose failed.");
+      // try next one
```

Comment:
You can add the same comment to `POPART_COUT("[poseEstimation]\tResection FAILED");` and remove the uppercase.